spec/lib/action_spec.rb

```diff
@@ -54,16 +54,27 @@
     end
   end
 
-  describe '#fetch_content' do
+  describe '#fetch_content_and_blob_sha' do
```

Comment:
If you have divided in two methods I would create two describe block one for each method for `fetch_content` and `blob sha`

Reply:
Done